packages/map-template/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@mapsindoors/map-template",
-  "version": "0.1.0",
+  "version": "1.0.0-rc.1",
```

Comment:
What is the `0-rc.1` for?

Reply:
It's a pre-release identifier. "rc" is for "release candidate".

https://semver.org/spec/v2.0.0-rc.1.html#spec-item-12